kolibri/plugins/style_guide/assets/src/views/content/api_table.vue

```diff
@@ -0,0 +1,101 @@
+<template>
```

Comment:
This component might be better named "api tables" plural since it's actually three tables.

Alternatively, consider breaking it into three components (without the `<h3>` tags): `props-table`, `slots-table`, and `events-table`.